### PR TITLE
Modified styless.css and index.html to reflect corrections

### PR DESCRIPTION
**Pull request by EddxSotz**
I made use of css grid and flexbox to position items for the Project section and flexbox for the About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,9 +54,84 @@ <h5 >I help bring awesome web experiences to life, if you have a project in mind
...
+                    </div>
+                    <img src="images/Img_Placeholder.png" alt="">
+                </div>
+                <div class="work-1">
+                    <div class="card_content">
+                        <h3>Website Portfolio</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore </p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
```
Comment (Favourezeugwa) [inline index.html]: - For some reason, your project cards are not been rendered properly in the project container. Can we kindly fix that ? 

Here is what I could see 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| Screenshot 2023-02-18 at 07 34 58 |Screenshot 2023-02-18 at 07 40 01  |

File: css/styles.css [hunk 2]
```diff
@@ -197,5 +203,190 @@ a{
...
+}
+
+/* About me Section*/
+#About{
+    background-color: #262626;
+    height: fit-content;
+    padding: 8rem 2rem;
+    position: relative;
+    overflow: hidden;
+}
```
Comment (Favourezeugwa) [inline css/styles.css]: - The font size of your `about-me` section seems to be a bit too large on a mobile screen. Can we reduce it to suit that on Figma. 

Here is what I can see 👇🏽 

| Your Design  | Figma Design |
| :---:   | :---: | 
| Screenshot 2023-02-18 at 07 42 42 |Screenshot 2023-02-18 at 07 49 13  |

File: css/styles.css [hunk 3]
```diff
@@ -197,5 +203,190 @@ a{
...
+/*#image-2{
+    position: absolute;
+        max-width: 60%;
+        right: -250px;
+        top: 100px;
+}*/
```
Comment (Favourezeugwa) [inline css/styles.css]: - I'd like to kindly suggest that you removed all commented lines of code to make your code base cleaner. 🌸

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @EddxSotz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -54,9 +54,85 @@ <h5 >I help bring awesome web experiences to life, if you have a project in mind
...
+                        <h3>Brand website</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore</p>
+                        <ul id="tags"><li>css</li><li>html</li><li>bootstrap</li><li>ruby</li></ul>
+                        
+                    </div>
+                </div>
+                <div class="work-4">
+                    <div class="card_content">
+                        <h3>Shopping Website</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore</p>
```
Comment (algerina) [inline index.html]: - [x] You did great implementing the changes required by the previous reviewer however, the boutons to show the projects are not displayed. Kindly fix that issue.
review

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @EddxSotz  ,

Good job so far!  But still some minor issues to complete.

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- No linters errors:heavy_check_mark:



## STATUS: REQUIRED CHANGES 




 ##required changes:

- Please check the comments under the review

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 4 above)
Comment (EddxSotz) [reply index.html]: @algerina Thank you, buttons have been added

**Review by Olamarx — APPROVED** [review]
Hi @EddxSotz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Descriptive PR ✔️ 
- commit messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**